webserver/webserver/src/test/java/io/helidon/webserver/utils/SocketHttpClient.java

```diff
@@ -382,6 +405,37 @@ public void request(String method, String path, String protocol, String host, It
         pw.flush();
     }
 
+    /**
+     * Send supplied text manually to socket.
+     * @param requestString text to send
+     * @param args format arguments
+     * @return this http client
+     * @throws IOException
+     */
+    public SocketHttpClient manualReq(String requestString, Object... args) throws IOException {
```

Comment:
I am not in favor of this method in general.
If it should exist, its name should be `manualRequest`, and the first parameter should be called `formatString` or similar.
Also please separate first `@param` with a new line from javadoc text.